frontend/src/features/mapSideBar/SidebarContents/subforms/ClassificationForm.test.tsx

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import { ClassificationForm } from './ClassificationForm';
+import renderer from 'react-test-renderer';
+import { useFormikContext, getIn } from 'formik';
+import { CLASSIFICATIONS } from 'constants/classifications';
+import { cleanup, render } from '@testing-library/react';
+
+jest.mock('formik');
+
+const form = <ClassificationForm classifications={[]} field="classificationId" />;
+
+(useFormikContext as jest.Mock).mockReturnValue({
+  values: {},
+  registerField: jest.fn(),
+  unregisterField: jest.fn(),
+});
+(getIn as jest.Mock).mockReturnValue(CLASSIFICATIONS.CoreOperational);
+
+it('renders correctly', () => {
+  const tree = renderer
+    .create(<ClassificationForm classifications={[]} field="classificationId" />)
+    .toJSON();
+  expect(tree).toMatchSnapshot();
+});
+
+describe('field definition conditional render', () => {
+  it('renders defintion when provided', () => {
+    const { getByText } = render(
+      <ClassificationForm
+        classifications={[]}
+        field="classificationId"
+        fieldDescription="test description"
+      />,
+    );
+    expect(getByText('test description')).toBeInTheDocument();
+  });
+});
+
+describe('renders definitions correctly', () => {
+  afterEach(() => {
+    cleanup();
+    jest.clearAllMocks();
+  });
+  it('renders default defintion', () => {
+    (getIn as jest.Mock).mockReturnValue('');
+    const { getByText } = render(form);
+    expect(getByText(/select a classification/i)).toBeInTheDocument();
+  });
+  it('renders core operational defintion', () => {
+    (getIn as jest.Mock).mockReturnValue(CLASSIFICATIONS.CoreOperational);
```

Comment:
I'd have a bit of a preference for triggering this change by a click instead of via mocking, but obviously time permitting.